Apps/ClinicalDocument/src/Api/IClinicalDocumentsApi.cs

```diff
@@ -28,9 +28,18 @@ public interface IClinicalDocumentsApi
         /// <summary>
         /// Retrieves clinical document records by patient identifier.
         /// </summary>
-        /// <param name="pid">The pid to get clinical document records.</param>
+        /// <param name="pid">The patient id to get clinical document records.</param>
```

Comment:
Nit: Are we sure we know that pid = patient id and not personal id?

Reply:
Yes according to the PatientIdentity model:

![Screen Shot 2022-09-14 at 11 17 04 AM](https://user-images.githubusercontent.com/5408101/190232164-ac801b21-9cff-483e-b41b-16461fc7191b.png)